pkg/scheduler/algorithm/priorities/requested_to_capacity_ratio.go

```diff
@@ -109,10 +128,21 @@ func buildRequestedToCapacityRatioScorerFunction(scoringFunctionShape FunctionSh
 		return rawScoringFunction(maxUtilization - (capacity-requested)*maxUtilization/capacity)
 	}
 
-	return func(requested, allocable *schedulernodeinfo.Resource, includeVolumes bool, requestedVolumes int, allocatableVolumes int) int64 {
-		cpuScore := resourceScoringFunction(requested.MilliCPU, allocable.MilliCPU)
-		memoryScore := resourceScoringFunction(requested.Memory, allocable.Memory)
-		return (cpuScore + memoryScore) / 2
+	return func(requested, allocable ResourceToValueMap, includeVolumes bool, requestedVolumes int, allocatableVolumes int) int64 {
+		var nodeScore, weightSum int64
+		nodeScore = 0
+		weightSum = 0
```

Comment:
please remove the initialization, by design ints are initialized to 0 in golang.

Reply:
👍